Content.Server/StationRecords/Systems/StationRecordsSystem.cs

```diff
@@ -71,7 +72,11 @@ private void OnPlayerSpawn(PlayerSpawnCompleteEvent args)
             return;
         }
 
-        CreateGeneralRecord(station, idUid.Value, profile.Name, profile.Age, profile.Species, profile.Gender, jobId, profile, records);
+        if (!EntityManager.TryGetComponent(player, out FingerprintComponent? fingerprintComponent))
```

Comment:
this should be `TryComp<FingerprintComponent>(player, out var prints)` then just pass pass `prints?.Fingerprint`, no need to abort the record if it has no prints